qml/Sailfish/Silica/ListItem.qml

```diff
@@ -9,8 +9,23 @@ SimpleRow {
 
     height: contentHeight
 
-    // HACK: Display arrow on the bottom right
-    text: " "
-    detailText: " "
+    // Do not show right angle in iOS
```

Comment:
"Angle", really? I know, the v-play guys use that term, as well.
But I think "arrow" would be the better term ;).